modules/codegen-plugin/src/smithy4s/codegen/Smithy4sCodegenPlugin.scala

```diff
@@ -278,7 +272,6 @@ object Smithy4sCodegenPlugin extends AutoPlugin {
               file,
               s"""$$version: "2"
                  |metadata smithy4sWildcardArgument = "$wildcardArg"
```

Comment:
thought: I would consider dropping `smithy4sWildcardArgument` too, we should use `*` and ask people to enable `-Xsource:3` or `-Xsource:3-cross`.